appinfo/preupdate.php

```diff
@@ -5,7 +5,7 @@
 	$query = OC_DB::prepare( 'SELECT COUNT(*) AS `count`, `fileid` FROM `*PREFIX*files_antivirus` GROUP BY `fileid` HAVING COUNT(*) > 1' );
 	$result = $query->execute();
 	while( $row = $result->fetchRow()) {
-		$deleteQuery = OC_DB::prepare('DELETE FROM `*PREFIX*files_antivirus` WHERE `fileid` = ? ORDER BY `check_time` ASC', $row['count']-1);
+		$deleteQuery = OC_DB::prepare('DELETE FROM `*PREFIX*files_antivirus` WHERE `fileid` = ?', $row['count']-1);
```

Comment:
@LukasReschke correct. Adding a PK it appeared that table might have several scan results if the app is old enough. This cleans up scan results except the very last.